src/NuGet.Clients/NuGet.Console/PowerConsoleToolWindow.cs

```diff
@@ -524,8 +524,6 @@ private async Task OnDispatcherStartCompletedAsync()
             // force the UI to update the toolbar
             IVsUIShell vsUIShell = await AsyncServiceProvider.GlobalProvider.GetServiceAsync<IVsUIShell>();
             vsUIShell.UpdateCommandUI(0 /* false = update UI asynchronously */);
-
-            NuGetEventTrigger.Instance.TriggerEvent(NuGetEvent.PackageManagerConsoleLoaded);
```

Comment:
I am curious to understand, why this line of code is unreachable. Is it because `vsUIShell.UpdateCommandUI` is invoked asynchronously?

[doc](https://docs.microsoft.com/en-us/dotnet/api/microsoft.visualstudio.shell.interop.ivsuishell.updatecommandui?view=visualstudiosdk-2019#Microsoft_VisualStudio_Shell_Interop_IVsUIShell_UpdateCommandUI_System_Int32_) says `If the method succeeds, it returns S_OK. If it fails, it returns an error code.` I am also not why we didn't capture the result of `UpdateCommandUI` method. Any idea?

Reply:
@kartheekp-ms 
End of line it tries to load method from `EventTrigger.dll`. But this dll doesn't exist anymore anywhere, so it'll never exercised.

https://github.com/NuGet/NuGet.Client/blob/f24bad0668193ce21a1db8cabd1ce95ba509c7f0/src/NuGet.Core/NuGet.PackageManagement/Utility/NuGetEventTrigger.cs#L29-L36